website/blog/2021-08-20-immutable-data-lakes.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Immutable data lakes using Apache Hudi"
+excerpt: "How to leverage Apache Hudi for your immutable (or) append only data use-case"
+author: shivnarayan
+category: blog
+---
+
+Apache Hudi helps you build and manage data lakes with different table types, config knobs to cater to everyone's need.
+We strive to listen to community and build features based on the need. From our interactions with the community, we got 
+to know there are quite a few use-cases where Hudi is being used for immutable or append only data. This blog will go 
+over details on how to leverage Apache Hudi in building your data lake for such immutable or append only data.
+
```

Comment:
@nsivabalan - Can you please add a <!--truncate--> tag here, so that the main blog page will render only up to this tag with a read more link?

When we don't have the truncate tag, the full blog post will be rendered on the blog homepage.